container_files/install.sh

```diff
@@ -15,6 +15,8 @@ git checkout mptcp_udp_retry_cherrypick
 echo "================== Successfully installed iperf  ======================"
 
 echo "================== Installing neper  ======================"
+mkdir -p /root/
+cd /root/
 git clone -b lnst-production https://github.com/LNST-project/neper.git
```

Comment:
`git clone`'s last argument is destination path (if provided). So it would be easier just to run:
```
git clone -b lnst-production https://github.com/LNST-project/neper.git /root/neper
```
It also creates all the missing parent directories along the path.

Anyway, not a big deal, so feel free to resolve and merge.

Reply:
> It also creates all the missing parent directories along the path.

ah, didn't try that... i'll test and fix.

Reply:
yup, that worked. thanks, resolving.